Assets/BossRoom/Scripts/Client/UI/Lobby/CreateLobbyUI.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Text;
+using BossRoom.Scripts.Shared.Infrastructure;
+using Unity.Multiplayer.Samples.BossRoom;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BossRoom.Scripts.Client.UI
+{
+    public class CreateLobbyUI : MonoBehaviour
+    {
+        public const string k_DefaultIP = "127.0.0.1";
```

Comment:
pretty sure we already defined this somewhere. It might be worth it to have a centralized constant utility, something like that.

Reply:
This declaration traveled from PopupPanel class to this one. I actually think that they belong here, since they are the UI defaults only. They don't touch any other logic.